extensions/cms/src/apiWrapper.ts

```diff
@@ -106,7 +108,7 @@ export class ApiWrapper {
 	}
 
 	public async setConfiguration(value: any): Promise<void> {
-		await vscode.workspace.getConfiguration('cms').update('cmsServers', value, true);
+		await vscode.workspace.getConfiguration('cms').update('servers', value, true);
```

Comment:
I wonder if the settings should be under a node `centralManagementServers`?  Do we use acronyms for other settings?

Reply:
Yeah - the section name definitely should be updated. Right now it's Cms: Cms Servers which isn't great at all

---
In reply to: [289213200](https://github.com/microsoft/azuredatastudio/pull/5542#discussion_r289213200) [](ancestors = 289213200)